### ISSUE TITLE
Can't delete a workspace - Website

### ISSUE THREAD

**Issue by AlissonRS** [a85/POSTMan-Chrome-Extension#6160]
Please read through the [guidelines](https://github.com/postmanlabs/postman-app-support#guidelines-for-reporting-issues) before creating a new issue.

**Describe the bug**
I created a team workspace and now I can't delete it. I searched other issues here and none of them seemed to be my case. When I try to delete it, it says:

> Some members might lose access to the collections and environments in this workspace. To avoid this, add them to another workspace before deleting this one.

However I'm already in another workspace (a personal one), so I don't understand why I get this message. I'm the only member in the workspace, and I can't leave/remove myself from it. So I can't delete the workpace.

**Screenshots**
![image](https://user-images.githubusercontent.com/4753901/55037907-fb1fdb80-4ffd-11e9-9360-c3e1b2cbb734.png)
![image](https://user-images.githubusercontent.com/4753901/55037934-0d017e80-4ffe-11e9-99a0-eceeee244e9e.png)

**App information (please complete the following information):**
 - App Type [e.g. Chrome App, Native App]
 - Postman Version [e.g. 6.0.1]
 - OS: [e.g. macOS High Sierra 10.13.2]

**Comment by preethammavin** [comment]
Hi @AlissonRS 
Can you please write to us at help@getpostman.com, so that we have more information to solve this issue.

**Comment by AlissonRS** [comment]
Hey @preethammavin thanks for your time. I sent an email, feel free to ask any further details that I missed which could help.

**Comment by preethammavin** [comment]
@AlissonRS thanks for the information, will give you an update when we have a resolution.

**Comment by preethammavin** [comment]
@AlissonRS I was able to check that you have 1 team workspace, and 2 personal ones. 

The one that you were trying to delete was the team default one, which postman creates for you which cant be deleted. This is in reference to the workspace id you provided which could not be deleted.

The default workspace is created to help the user onboard, we don't let them delete it purely to ensure they are able to understand the product completely. 

I understand the error message should be better that what you were shown, we are working on it. If you want to move out of the team context itself, you can do so by leaving the team at the team setting page in the web dashboard https://go.postman.co/settings/me/team. 

Do let me know if this doesnt solve the problem

**Comment by AlissonRS** [comment]
@preethammavin thanks for your reply, it was sooner than I expected.

It's weird as the 1 team workspace I had (whose name was **Meds** as displayed in my screenshots) I created myself from the Postman desktop client, it was not a workspace that existed since my account was created and I just renamed it to Meds.

Another odd thing is that I wasn't being able to leave the team from the screen I printed and pasted here, however by using the link you provided I was able to leave the team, and the team was automatically deleted (since I was the last member).

I really appreciate your attention and thanks for considering improving the messages.

Cheers!!!
